languages/gutenberg.pot

```diff
@@ -3,15 +3,15 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "X-Generator: babel-plugin-wp-i18n\n"
 
-#: editor/blocks/freeform/index.js:4
```

Comment:
See also: #377

We could probably skip these changes. I'll hope to tackle this in the next day.